keras_cv/layers/preprocessing/jittered_resize.py

```diff
@@ -193,7 +193,7 @@ def augment_bounding_boxes(self, bounding_boxes, transformation, **kwargs):
                 "Please provide a `bounding_box_format` when augmenting "
                 "bounding boxes with `JitteredResize()`."
             )
-
+        result = bounding_boxes.copy()
```

Comment:
instead of doing copy, can we just say result = bounding_boxes, then later:
`
boxes = bounding_boxes.pop("boxes")
boxes = convert_format(boxes, ...)
result["boxes"] = boxes
`
?

Reply:
.copy() only copies the dictionary - we aren't copying a ton of Tensors, we're copying a single dictionary during trace time.   I don't think this is a concern performance wise.